OrbitGl/App.h

```diff
@@ -208,6 +209,9 @@ class OrbitApp final : public DataViewFactory, public CaptureListener {
   [[nodiscard]] int32_t GetSelectedProcessID() const {
```

Comment:
ok, is this used anywhere? should it be renamed?

Reply:
Yes, and maybe, but may not.